content/templates/_components/c-button-skins/_docs.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Buttons (skins)
+title: Buttons
```

Comment:
Should you also change the directory name ? (This is `c-button-skins`, i.e. with the `skins` suffix.)

Reply:
Correct, should be adjusted now! 24b58bddbf189379487b5f3dc58166667e33142c